src/js/shapes.js

```diff
@@ -15,11 +15,11 @@ var Shapes = function Shapes(settings) {
   this.active = true;
 
   var self = this,
-    glLayer = this.glLayer = canvasOverlay(function() {
+    glLayer = this.glLayer = canvasOverlay(function() { // creates gl  layr
```

Comment:
If you'd like, you could move this function into a method, so it self documents.